File: Algorithms/BOJ/Python/11328.py
# https://www.acmicpc.net/problem/11328
# Strfry
import sys

n = int(sys.stdin.readline())
for _ in range(n):
    a, b = sys.stdin.readline().split()
    if sorted(a) == sorted(b):
        print("Possible")
    else:
        print("Impossible")

File: Algorithms/BOJ/Swift/11328.swift
// https://www.acmicpc.net/problem/11328
// Strfry

let n = Int(readLine()!)!

for _ in 0..<n {
    let word = readLine()!.split(separator: " ").map { String($0) }
    if word[0].sorted() == word[1].sorted() {
        print("Possible")
    } else {
        print("Impossible")
    }
}
